src/statd/statd.c

```diff
@@ -391,6 +391,31 @@ static int ly_add_ip_link_br(const struct ly_ctx *ctx, struct lyd_node **parent,
 	return SR_ERR_OK;
 }
 
+static int iface_has_flag(json_t *j_iface, const char *ifflag)
```

Comment:
If you want to align the variable naming to the rest of the statd code, this should be `const char *flag` and the json flags below should be `j_flags`and `j_flag`. No big deal.

Reply:
OK, will fix!